testing/how-to-launch-tests.md

```diff
@@ -47,3 +47,25 @@ vendor/bin/phpunit --debug -c tests/Unit/phpunit.xml tests/Unit/PrestaShopBundle
 {{% notice tip %}}
 Hint: `--debug` without `--filter` gives a nice list to filter with.
 {{% /notice %}}
+
+# Checking for code coverage in phpunit tests
+
+To check if you covered everything in your tests cases its best to run the tests with coverage report.
+
+First you have to enable xdebug. For the docker dev environment run
+
+```
+docker-php-ext-enable xdebug
```

Comment:
Why talk about docker? :thinking:

Reply:
yeah, maybe this should have a link to this page: https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/contribute_using_docker/#how-to-become-a-core-contributor-using-docker (and that page to have a link back to here)

Reply:
and also, it's when you're inside the container, for someone who don't know docker, docker-compose, he will not understand anything about this sentence

Reply:
```
docker-compose exec prestashop-git XDEBUG_MODE=coverage vendor/bin/phpunit --coverage-text -c tests/Unit/phpunit.xml tests/Unit/PrestaShopBundle/Command/ConfigCommandTest.php 
```

would be nice but I cant get it working (probably just something stupid but I cannot see it)

```
OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "XDEBUG_MODE=coverage": executable file not found in $PATH: unknown
```

Reply:
aah, of course .. the env variable needs to be pushed with -e

```
docker-compose exec -e XDEBUG_MODE=coverage prestashop-git vendor/bin/phpunit --coverage-text -c tests/Unit/phpunit.xml tests/Unit/PrestaShopBundle/Command/ConfigCommandTest.php 
```

seems to work correctly

I'll update in a second

Reply:
@PierreRambaud see, I'm learning :)